index.html

```diff
@@ -1,62 +1,90 @@
+    <link rel="stylesheet" href="CSS/style.css" />
+    <title>Hack Your Future Belgium</title>
+  </head>
+  <body>
+    <!--Header-->
+    <header>
+      <div class="header-container">
+        <!--Logo-->
+        <div class="hyf-logo">
+          <a href="index.html">
+            <img
+              class="logo"
+              src="IMAGES/logo-HYF.svg"
+              alt="HackYourFuture logo"
+            />
+          </a>
+        </div>
+        <!--Navigation section of the header: contains related links-->
 
-            <div class="nav-menu">
-               <nav>
-                  <ul class="links">
-                     <li><a href="program.html">program</a></li>
-                     <li><a href="#"></a>about</li>
-                     <li><a href="#"></a>volunteer</li>
-                     <li><a href="#"></a>support us</li>
-                     <li><a href="#"></a>faq</li>
-                  </ul>
-               </nav>
+        <div class="nav-menu">
+          <nav>
+            <ul class="links">
+              <li><a href="program.html">program</a></li>
+              <li><a href="#"></a>about</li>
+              <li><a href="#"></a>volunteer</li>
+              <li><a href="#"></a>support us</li>
+              <li><a href="#"></a>faq</li>
+            </ul>
+          </nav>
 
-               <!--Button Apply Now-->
-               <div class="btn-apply">
-                  <a
-                     href="https://docs.google.com/forms/d/e/1FAIpQLSfS0215I6BCy1jzpWmxv9UxYxp5I25ZcoWhSxijhcUImLELcg/viewform"
-                     target="_blank"
-                     class="apply-button"
-                     >apply now</a
-                  >
-               </div>
+          <!--Button Apply Now-->
+          <div class="btn-apply">
+            <a
+              href="https://docs.google.com/forms/d/e/1FAIpQLSfS0215I6BCy1jzpWmxv9UxYxp5I25ZcoWhSxijhcUImLELcg/viewform"
+              target="_blank"
+              class="apply-button"
+              >apply now</a
+            >
+          </div>
+        </div>
+      </div>
+    </header>
+    <!--Main-->
+    <main>
+      <!--Our program-->
+      <section class="our-program">
+        <div class="op-container-2">
+          <div class="op-container-3">
+            <!-- Left Container -->
+            <div class="our-program-left">
+              <h3>Our program</h3>
+              <p>
+                Our <b>8-month</b> program is divided into <b>12 modules</b>.
+                The last module is a
+                <b>6-weeks intensive Final Project</b> where students work on a
+                real project with a real partner. Our training takes place at
+                <b>BeCentral</b> (Brussels) every <b>Sunday</b>. During the
+                week, students are supported online in their <b>self-study</b>,
+                homework and peer learning.
+              </p>
+              <button type="button">Discover more</button>
```

Comment:
That's a good point. I made my button as a link, so I will remake one.

Reply:
@katsmamina it is not a problem to make a button with a tag, you can style it to look like a button. I think Jose forgot to add the link to the program page here.

Reply:
Yes. There is no link. I will modify it.
And regardig the button. As Rayana saids it is an option to anchor a button into a link. I saw that in Our program section of the HYF site when i codereversed it.